backend/stackdriver.go

```diff
@@ -3,29 +3,34 @@ package backend
 import (
 	"context"
 	"fmt"
+	"log"
+	"strings"
+	"time"
+
 	"github.com/buildkite/buildkite-agent-metrics/collector"
 	"github.com/golang/protobuf/ptypes/timestamp"
 	"google.golang.org/genproto/googleapis/api/label"
-	"log"
-	"time"
 
-	"cloud.google.com/go/monitoring/apiv3"
+	monitoring "cloud.google.com/go/monitoring/apiv3"
 	"google.golang.org/genproto/googleapis/api/metric"
 	monitoringpb "google.golang.org/genproto/googleapis/monitoring/v3"
 )
 
 const (
-	metricTotalPrefix = "custom.googleapis.com/buildkite/total/%s"
-	queueLabelKey = "Queue"
-	queueDescription = "Queue Descriptor"
+	metricName        = "custom.googleapis.com/buildkite/%s/%s"
```

Comment:
I get the addition of a placeholder for org here - whats the thought process on changing the naming "total" here?

Reply:
I thought the "total" somehow signified that the mixing of the data of all organizations was intentional (otherwise I'm not sure what "total" means here - total of what?). Thus, I thought now that we split the data per org (and it's no longer a "total"), we just put the org's name in the place of the former "total", because we no longer need it.

But: If you feel like it should still start with "total", I guess I just didn't understand the meaning behind it - please explain! :)